client/components/repository/WorkflowBoard/Sidebar/index.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <v-navigation-drawer
+    width="450"
+    color="grey lighten-5"
+    absolute right permanent>
+    <article
+      v-if="selectedTask"
+      :key="selectedTask.uid"
+      class="px-4 pt-4 pb-10">
+      <sidebar-header v-bind="selectedTask" />
+      <task-field-group @update="updateTask" v-bind="selectedTask" class="mt-9 mb-4" />
+      <section>
```

Comment:
@underscope should we move it above or below the task title?

Reply:
If it's above the task title, it might have a negative effect on the user's ability to associate the selected card with the sidebar data.

If it's below the task title, the form section looks like it's part of related content. We might be able to help this by adding a title to the form section, but it's inconsistent with the sidebar's header and the rest of the app's UI.

@underscope thoughts?

<img width="400" alt="Screen Shot 2020-10-21 at 01 27 23" src="https://user-images.githubusercontent.com/9119913/96655147-191a0300-133d-11eb-878b-5f9756da9a9c.png">

Reply:
It might help if instead of the card we just put the link to the content item with details on hover (or popover with an info icon) and make label a bit more discrete. Thoughts 🤔 ?
Also, one thing to consider, what if I just want to assign the content item to the author with priority (without task name/title, should we allow that ?)

Reply:
If the task name is the primary way for the user to identify a task, we'll lose one step in the task creation, but also reduce the user's ability to quickly scan the content. We'd also blur the line between task and activity which might be preferred, but would require tweaking the rest of the UI to communicate the intent to the user.
Would it be worth to think about it when we plan the author-oriented view we talked about?